src/plots/gl3d/scene.js

```diff
@@ -58,10 +58,10 @@ function render(scene) {
     }
 
     function formatter(axisName, val) {
-        if(typeof val === 'string') return val;
-
         var axis = scene.fullSceneLayout[axisName];
-        return Axes.tickText(axis, axis.c2l(val), 'hover').text;
+        var calendar = trace[axis._id.charAt(0) + 'calendar'];
```

Comment:
So, if I understand correctly: hover labels should show dates with the calendar of the axes (as opposed to the calendar of the trace)?